sympy/matrices/matrices.py

```diff
@@ -3015,6 +3015,8 @@ def berkowitz_minors(self):
 
         berkowitz
         """
+        if not self:
```

Comment:
Is this test necessary after ``berkowitz`` has been modified to work with an empty matrix?